lib/registrant.rb

```diff
@@ -0,0 +1,40 @@
+# We should also be able to change a permit from false to true after a Registrant has earned their permit. 
+# There is no age limit to when someone can earn their permit.
+
+require 'pry'
```

Comment:
Because you have these gems in your Gemfile, you shouldn't need to require them in every file as long as you run `bundle install` when setting up the repo